integration/init/cockroach/expected/base/Service-cockroachdb-public.yaml

```diff
@@ -1,21 +1,16 @@
 apiVersion: v1
 kind: Service
 metadata:
-  # This service is meant to be used by clients of the database. It exposes a ClusterIP that will
-  # automatically load balance connections to the different database pods.
-  name: cockroachdb-public
   labels:
     app: cockroachdb
+  name: cockroachdb-public
 spec:
   ports:
-  # The main port, served by gRPC, serves Postgres-flavor SQL, internode
-  # traffic and the cli.
-  - port: 26257
+  - name: grpc
+    port: 26257
     targetPort: 26257
-    name: grpc
-  # The secondary port serves the UI as well as health and debug endpoints.
-  - port: 8080
+  - name: http
+    port: 8080
     targetPort: 8080
-    name: http
   selector:
-    app: cockroachdb
\ No newline at end of file
+    app: cockroachdb
```

Comment:
I don't really like how we're losing comments here
I suppose there's nothing for it though
Unless it's possible to use Ethan's comment preservation code... But I really highly doubt it

Reply:
it might actually - i'll give it a go